addOns/wappalyzer/src/test/java/org/zaproxy/zap/extension/wappalyzer/PassiveScannerUnitTest.java

```diff
@@ -70,18 +72,22 @@ protected WappalyzerPassiveScanner createScanner() {
         return new WappalyzerPassiveScanner(getDefaultHolder());
     }
 
-    @Test
-    void testApacheWithPhp() throws HttpMalformedHeaderException {
+    @ParameterizedTest
+    @CsvSource({"false, ''", "true, 5.6.34"})
+    void testApacheWithPhp(boolean exhaustive, String phpVersion)
```

Comment:
Worth adding a new specific test, just to have a more appropriate name to what is actually being tested.

Reply:
It could have kept it parameterized, to not test the behaviour in two methods.